lib/foreman_maintain/param.rb

```diff
@@ -3,7 +3,8 @@ class Param
     attr_reader :name, :description, :options
 
     def initialize(name, description, options, &block)
-      options.validate_options!(:description, :required, :flag, :array, :allowed_values, :default)
+      options.validate_options!(:description, :required, :flag, :array,
+                                :allowed_values, :default, :hidden)
```

Comment:
Shouldn't be here when hidden was removed

Reply:
`:hidden` i still present

Reply:
Done!